docs: update debugging-uno-ui.md

jonpryor opened the pull request:
Elaborate on the changes that should be made to `global.json` if you want an external app to use your unoplatform/uno changes, and how to obtain the `$(UnoNugetOverrideVersion)` value.

Remove mention of `Uno.Sdk.Private`, as it doesn't appear to be needed for the described workflow.

**GitHub Issue:** closes #

<!-- Link to relevant GitHub issue if applicable. All PRs should be associated with an issue (GitHub issue or internal), unless the change is documentation related. -->

## PR Type:

<!--
Copy the labels that apply to this PR and paste them above:

- 🐞 Bugfix
- ✨ Feature
- 🎨 Code style update (formatting)
- 🔄 Refactoring (no functional changes, no api changes)
- 🏗️ Build or CI related changes
- 📚 Documentation content changes
- 🤖 Project automation
- 💬 Other... (Please describe)

-->


## What is the current behavior? 🤔

<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->


## What is the new behavior? 🚀

<!-- Please describe the new behavior after your modifications. -->

## PR Checklist ✅

Please check if your PR fulfills the following requirements:

- [ ] 📝 Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.
- [ ] 🧪 Added [Runtime tests, UI tests, or a manual test sample](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] 📚 Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] 🖼️ Validated PR `Screenshots Compare Test Run` results.
- [ ] ❗ Contains **NO** breaking changes

<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below.
     Please note that breaking changes are likely to be rejected -->

## Other information ℹ️

<!-- Please provide any additional information if necessary -->

jonpryor committed:
docs: update debugging-uno-ui.md

Elaborate on the changes that should be made to `global.json` if
you want an external app to use your unoplatform/uno changes,
and how to obtain the `$(UnoNugetOverrideVersion)` value.

Remove mention of `Uno.Sdk.Private`, as it doesn't appear to be
needed for the described workflow.

Xiaoy312 commented on doc/articles/uno-development/debugging-uno-ui.md:
````diff
@@ -24,23 +24,55 @@ See [this article](working-with-the-samples-apps.md) for more information on wor
+First, update `global.json` of the app you will be debugging to instead use a `-dev.xxx` version:
+
+```diff
+diff --git a/global.json b/global.json
+index 509ebab..a2bb2f7 100644
+--- a/global.json
++++ b/global.json
+@@ -1,9 +1,9 @@
+ {
+   // To update the version of Uno please update the version of the Uno.Sdk here. See https://aka.platform.uno/upgrade-uno-packages for more information.
+   "msbuild-sdks": {
+-    "Uno.Sdk": "6.1.23"
++    "Uno.Sdk": "6.2.0-dev.59"
+   },
````
dont need the full diff here, this should be enough
```suggestion
 {
   // To update the version of Uno please update the version of the Uno.Sdk here. See https://aka.platform.uno/upgrade-uno-packages for more information.
   "msbuild-sdks": {
-    "Uno.Sdk": "6.1.23"
+    "Uno.Sdk": "6.2.0-dev.59"
   },
```
we mentioned about the file name in previous line, so that is redundant too

unodevops commented:
⚠️⚠️ The build [172769](https://dev.azure.com/uno-platform///_build/results?buildId=172769) has failed on Uno.UI - docs.

jonpryor committed:
docs: Address feedback

Thanks to @Xiaoy312 and @agneszitte!

unodevops commented:
⚠️⚠️ The build [172773](https://dev.azure.com/uno-platform///_build/results?buildId=172773) has failed on Uno.UI - docs.

jonpryor committed:
docs: appease the Markdown linter

The Markdown Linter used does not support `***bolded italic***`;
remove that use from the note.

The Markdown Linter also wants `_` for italic and `**` for bold;
adjust inputs to match requirements.

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-21198/docs/index.html

jonpryor committed:
docs: Provide Windows path example
